src/pages/settings/Payments/AddPayPalMePage.js

```diff
@@ -92,6 +114,9 @@ class AddPayPalMePage extends React.Component {
                                 placeholder={this.props.translate('addPayPalMePage.yourPayPalUsername')}
                                 onChangeText={text => this.setState({payPalMeUsername: text})}
```

Comment:
Please clear the error on the `onChangeText`

Reply:
done